src/arch/i386/Makefile.pcbios

```diff
@@ -58,19 +58,19 @@ NON_AUTO_MEDIA	+= iso
 %iso:	%lkrn util/geniso
 	$(QM)$(ECHO) "  [GENISO] $@"
 	$(Q)ISOLINUX_BIN=$(ISOLINUX_BIN) LDLINUX_C32=$(LDLINUX_C32) \
-	    VERSION="$(VERSION)" bash util/geniso -o $@ $<
+	    VERSION="$(VERSION)" bash -e util/geniso -o $@ $<
```

Comment:
`util/geniso` and `util/gensdsk` have been replaced with `util/genfsimg` which already includes a `set -e` within the script itself.